language/compiler/src/unit_tests/function_tests.rs

```diff
@@ -84,21 +84,3 @@ fn compile_script_with_large_frame() {
     let compiled_module_res = compile_module_string(&code);
     assert!(compiled_module_res.is_ok());
 }
-
-#[test]
```

Comment:
What did this test do exactly and why is it gone?

Reply:
The test is checking that the IR compiler errors when you have too many locals. It's now replaced by the new IR tests added.